README.rdoc

```diff
@@ -134,7 +134,14 @@ You can also directly access Product.elastic_index.mapping and Product.elastic_i
 
 Create the index:
 
-  rake index:create
+  rake index:create CLASS=Product
+
+If you need to create multiple indexes via the rake task, you will need to set the models in the
+global ElasticRecord configuration:
+
+  ElasticRecord.configure do |config|
+    config.model_names = %w(Product Order Location)
+  end
```

Comment:
The configuration docs have been moved up below the setup section, and more configuration options have been documented.